src/bundle/ui-dev/src/modules/universal-discovery/components/search/search.js

```diff
@@ -136,23 +157,26 @@ const Search = ({ itemsPerPage }) => {
             ];
 
             return (
-                <div className="c-search__no-results">
-                    <img className="" src="/bundles/ibexaadminui/img/no-results.svg" />
-                    <h2 className="c-search__no-results-title">{noResultsLabel}</h2>
-                    <div className="c-search__no-results-subtitle">
-                        {noResultsHints.map((hint, key) => (
-                            <div
-                                key={key} // eslint-disable-line react/no-array-index-key
-                                className="c-search__no-results-hint"
-                            >
-                                <div className="c-search__no-results-hint-icon-wrapper">
-                                    <Icon name="approved" extraClasses="ibexa-icon--small-medium" />
+                <>
+                    {renderCustomTableHeader()}
+                    <div className="c-search__no-results">
+                        <img className="" src="/bundles/ibexaadminui/img/no-results.svg" />
```

Comment:
```suggestion
                        <img src="/bundles/ibexaadminui/img/no-results.svg" />
```

Reply:
Hmm, this was in the original code I moved. I will delete it.